homeassistant/components/unifi_access/const.py

```diff
@@ -0,0 +1,5 @@
+"""Constants for the UniFi Access integration."""
+
+DOMAIN = "unifi_access"
+
+DEFAULT_HOST = "unifi.localdomain:12445"
```

Comment:
If that is indeed the URL we have to put in, please consider requesting the user to put in an URL and use YARL to explode that URL into bits you store to rebuild it later on. Check Overseerr for an example. Overseerr is also a self hosted service and thus asking an URL is easy as they can just copy paste what they have in their URL bar and the integration will figure out what the SSL setting, host, port and whatever is.

Reply:
Since UniFi consoles defaults their own name to unifi.localdomain and this service is available on port 12445 on such consoles I assume for mostly everyone that runs this on acutal UniFi hardware this default will also be the actual host used. Do we still think having the user fill out the value is better even though this should be a sensible default for most?

Reply:
Does that work via mDNS? I am not sure if that works for everyone, the users of this integration most likely already have unifi stuff, so they usually have a more special network setup

Reply:
Yeah its mDNS. The default configuration shipped with UniFi controllers advertise the gateway/controller as unifi.localdomain via mDNS.